ca_on_toronto/committees.py

```diff
@@ -4,8 +4,56 @@
 from .helpers import build_lookup_dict, committees_from_sessions
 from .constants import TWO_LETTER_ORG_CODE_SCHEME
 
+import re
+
+
+MEMBERS_URL_TEMPLATE = 'http://app.toronto.ca/tmmis/decisionBodyProfile.do?function=doGetMembers&meetingId={}&showLink=true'
+DEFAULT_COMMITTEE_ROLE = 'Member'
+
+REFERENCE_MEETING_IDS = {
```

Comment:
Actually, I can add a "term" layer to the hash. that's an easy way to address one of your points. one sec